src/components/timeline/TrackMemoryGraph.js

```diff
@@ -338,27 +348,43 @@ class TrackMemoryGraphImpl extends React.PureComponent<Props, State> {
   };
 
   _renderTooltip(counterIndex: number): React.Node {
-    if (this.props.accumulatedSamples.length === 0) {
+    const { accumulatedSamples, counter } = this.props;
+    if (accumulatedSamples.length === 0) {
       // Gecko failed to capture samples for some reason and it shouldn't happen for
       // malloc counter. Print an error and bail out early.
       throw new Error('No accumulated sample found for memory counter');
     }
-    const { minCount, countRange, accumulatedCounts } =
-      this.props.accumulatedSamples[0];
+    const { minCount, countRange, accumulatedCounts } = accumulatedSamples[0];
     const bytes = accumulatedCounts[counterIndex] - minCount;
+    const { number } = counter.sampleGroups[0].samples;
+    const allocations = number[counterIndex];
     return (
       <div className="timelineTrackMemoryTooltip">
         <div className="timelineTrackMemoryTooltipLine">
           <span className="timelineTrackMemoryTooltipNumber">
             {formatBytes(bytes)}
           </span>
-          {' relative memory at this time'}
+          <Localized id="TrackMemoryGraph--relative-memory-at-this-time">
+            relative memory at this time
+          </Localized>
         </div>
+
         <div className="timelineTrackMemoryTooltipLine">
           <span className="timelineTrackMemoryTooltipNumber">
             {formatBytes(countRange)}
           </span>
-          {' memory range in graph'}
+          <Localized id="TrackMemoryGraph--memory-range-in-graph">
+            memory range in graph
+          </Localized>
+        </div>
+
+        <div className="timelineTrackMemoryTooltipLine">
+          <span className="timelineTrackMemoryTooltipNumber">
+            {formatNumber(allocations, 2, 0)}
```

Comment:
I'm still a bit uncertain about the order of these fields. I wanted to group the fields that indicate bytes at the top and add this operation count to the bottom. But we could also group by the fields that change for every sample. Since the memory range (2nd currently) doesn't change at all, we can also put it to the bottom.

Reply:
Good question. Maybe putting the number of operations first would make more sense? What do you think?